wrapper/src/main/java/software/amazon/jdbc/dialect/AuroraMysqlDialect.java

```diff
@@ -83,4 +83,8 @@ public HostListProviderSupplier getHostListProvider() {
         NODE_ID_QUERY,
         IS_READER_QUERY);
   }
+
+  public static String getTopologyQuery() {
```

Comment:
you could check if the superclasses also need that function and define it there or just the subclasses

Reply:
Looks like the `TOPOLOGY_QUERY` field isn't defined in the superclass, just the subclasses